ClangSharpPInvokeGenerator/CursorWriter.cs

```diff
@@ -373,6 +383,38 @@ private bool BeginHandleDeclRefExpr(CXCursor cursor, CXCursor parent)
             }
         }
 
+        private bool BeginHandleDLLExport(CXCursor cursor, CXCursor parent)
+        {
+            Debug.Assert(cursor.Kind == CXCursorKind.CXCursor_DLLExport);
+
+            if (_attachedData.TryGetValue(parent, out var data) && (data is AttachedFunctionDeclData functionDeclData))
+            {
+                functionDeclData.IsDllExport = true;
+                return true;
+            }
+            else
+            {
+                Unhandled(cursor, parent);
+                return false;
+            }
+        }
+
+        private bool BeginHandleDLLImport(CXCursor cursor, CXCursor parent)
+        {
+            Debug.Assert(cursor.Kind == CXCursorKind.CXCursor_DLLImport);
+
+            if (_attachedData.TryGetValue(parent, out var data) && (data is AttachedFunctionDeclData functionDeclData))
+            {
+                functionDeclData.IsDllImport = true;
```

Comment:
We should probably look for this as a requirement to emitting the method in the generated code. If it isn't there, then it might have a method body (for example) and it won't resolve at runtime.